plugins/flytekit-kf-pytorch/tests/test_pytorch_task.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
should we remove this?

Reply:
ec68536f1

Thanks, surprised this was not automatically caught and fixed by the linter

Reply:
I saw all test folders are excluded from `ruff`:
```toml
extend-exclude = [
  "tests/",
  "**/tests/**",
]
```

I personally think it's better to ignore certain rules for test folders.